SpaceWarpMod/UI/ProcessArgumentsWindow.cs

```diff
@@ -0,0 +1,78 @@
+﻿using KontrolSystem.KSP.Runtime;
+using KontrolSystem.KSP.Runtime.KSPGame;
+using KontrolSystem.SpaceWarpMod.Core;
+using KSP.Game;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace KontrolSystem.SpaceWarpMod.UI {
+    public class ProcessArgumentsWindow : ResizableWindow {
+
+        private KontrolSystemProcess process;
+        private EntrypointArgumentDescriptor[] argumentDescs;
+        private string[] argumentValues;
+
+        public void Attach(KontrolSystemProcess process) {
+            var gameMode = GameModeAdapter.GameModeFromState(GameManager.Instance.Game.GlobalGameState.GetState());
+
+            argumentDescs = process.EntrypointArgumentDescriptors(gameMode);
+            argumentValues = argumentDescs.Select(arg => arg.DefaultValue.ToString()).ToArray();
+
+            this.process = process;
+            Open();
+        }
+
+        public void Awake() {
+            Initialize($"Program Arguments", new Rect(Screen.width - 750, Screen.height - 600, 0, 0), 120, 120, false);
```

Comment:
Maybe give it a tiny additional offset so that it does on aligns with the main window exactly.